app/views/admin/videos/new.html.haml

```diff
@@ -0,0 +1,21 @@
+%section.admin_add_video
+  .container
+    .row
+      .col-md-10.col-md-offset-1
+        = bootstrap_form_for [:admin, @video], layout: "form-horizontal" do |f|
```

Comment:
It should be `layout: :horizontal` or `layout: "horizontal"`, the "form-" portion is a remnant from when it was a css class.